examples/stress_tests/many_lights.rs

```diff
@@ -70,7 +73,38 @@ fn setup(
     }
 
     // camera
-    commands.spawn_bundle(PerspectiveCameraBundle::default());
+    let mut camera = match std::env::args().nth(1).as_deref() {
+        Some("orthographic") => {
+            let mut orthographic_camera_bundle = OrthographicCameraBundle::new_3d();
+            orthographic_camera_bundle.orthographic_projection.scale = 20.0;
+            let view_projection = orthographic_camera_bundle
+                .orthographic_projection
+                .get_projection_matrix();
+            orthographic_camera_bundle.frustum = Frustum::from_view_projection(
+                &view_projection,
+                &Vec3::ZERO,
+                &Vec3::Z,
+                orthographic_camera_bundle.orthographic_projection.far(),
+            );
+            commands.spawn_bundle(orthographic_camera_bundle)
+        }
+        _ => commands.spawn_bundle(PerspectiveCameraBundle::default()),
+    };
+
+    // NOTE: The lights in this example are distributed primarily in x and y across the
+    // screen. Using only 1 z-slice and letting the x,y slicing be calculated
+    // automatically produces much better light distribution across the clusters (i.e.
+    // fewer maximum lights in any one cluster). The consequences of this versus the
+    // default on an M1 Max means going from ~65fps to ~130fps!!!
+    camera.insert(ClusterConfig::FixedZ {
```

Comment:
On the one hand, I get why we're doing this. On the other hand, I think in practice what really matters is how the defaults behave. I'm guessing 99.9% of users won't think to touch these values.